ibis/backends/flink/__init__.py

```diff
@@ -384,12 +406,38 @@ def drop_table(
         )
         self._exec_sql(statement.compile())
 
+    def rename_table(
+        self,
+        old_name: str,
+        new_name: str,
+        must_exist: bool = True,
+    ) -> None:
+        """Rename an existing table.
+
+        Parameters
+        ----------
+        old_name
+            The old name of the table.
+        new_name
+            The new name of the table.
+        """
+        statement = RenameTable(
+            old_name=old_name,
+            new_name=new_name,
+            must_exist=must_exist,
+        )
+        sql = statement.compile()
+        self._exec_sql(sql)
+
     def create_view(
         self,
         name: str,
-        obj: ir.Table,
-        *,
+        obj: ir.Table | None = None,
+        query_expression: str | None = None,
         database: str | None = None,
+        catalog: str | None = None,
+        temporary: bool = False,
+        if_not_exists: bool = False,
```

Comment:
Where are all these extra arguments coming from? My understanding is that something like `create_view` would provide more-or-less the same interface as other backends.

Reply:
It would be helpful for me if you can point to a backend you may have referenced doing something like this off of; I'm just curious if other backends have different backend-specific `create_view` signatures, or just this one. I did see that the test code is unchanged, so that's good.

Reply:
> Where are all these extra arguments coming from?

I added the extra arguments according to the [doc](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/create/#create-view) for `create view`.

> It would be helpful for me if you can point to a backend you may have referenced doing something like this off of ...

I went over the definition of `create_view` for other backends, but could not spot any other with different signature.
I added the optional parameter `query_expression` per the [doc](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/create/#create-view) to make the `create_view` more flexible, but that parameter is currently not used. I can remove it to make it consistent with other backends. Please let me know.

Reply:
Yeah I'd vote for removing `query_expression`, at least for the time being anyway. For the other arguments, they sound reasonable to me because I also had to add some arguments that are not in other backends, but I'd just be cautious that the naming of these parameters are consistent across methods (e.g. `if_not_exists`, `must_exist`, etc).

Reply:
Thanks, removed `query_expression`.

> but I'd just be cautious that the naming of these parameters are consistent across methods (e.g. if_not_exists, must_exist, etc).

Good point. I spent some time contemplating on `if_not_exists` vs. `must_exist` vs. `force`. I decided to name these with "most explicit" names possible, but I guess here consistency across functions is more important than being explicit. This is why I now changed them to `force` in the related functions `create_view()`, `rename_table()`, `drop_table()`. Please let me know if you think otherwise.